transcendence-app/test/user/user.e2e-spec.ts

```diff
@@ -0,0 +1,95 @@
+import { HttpStatus, INestApplication, ValidationPipe } from '@nestjs/common';
+import * as request from 'supertest';
+import { ConfigModule } from '@nestjs/config';
+import { Test, TestingModule } from '@nestjs/testing';
+import { UserModule } from '../../src/user/user.module';
+import { v4 as uuidv4 } from 'uuid';
+import { validate } from '../../src/config/env.validation';
+import { UserController } from '../../src/user/user.controller';
+import { AuthenticatedGuard } from '../../src/shared/guards/authenticated.guard';
+import { UserDto } from '../../src/user/dto/user.dto';
+
+const testUserDto: UserDto = {
+  username: 'user',
+  email: 'afgv@github.com',
+  avatar_id: null,
+};
+const testUserId = uuidv4();
+
+describe('[Feature] User - /users', () => {
```

Comment:
These are the e2e tests from user.controller.spec.ts. I've moved them here because they require PostgreSQL.